pkgs/os-specific/linux/trezor-udev-rules/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "trezor-udev-rules";
+  version = "unstable-2019-07-17";
+
+  udevRules = fetchurl {
+    # let's pin the latest commit in the repo which touched the udev rules file
+    url = "https://raw.githubusercontent.com/trezor/trezor-firmware/54d348228f1187f356f3ae7f1fcc618243108121/common/udev/51-trezor.rules";
+    sha256 = "1q8ijd2jnkl16n7qipajipmggl8n7iqi88kdfwsx1152z2h1sn8k";
+  };
+
+  dontUnpack = true;
+
+  installPhase = ''
+    cp ${udevRules} 51-trezor.rules
+    mkdir -p $out/lib/udev/rules.d
+    # we use trezord group, not plugdev
+    # we don't need the udev-acl tag
+    substituteInPlace 51-trezor.rules \
+      --replace 'GROUP="plugdev"' 'GROUP="trezord"' \
+      --replace ', TAG+="udev-acl"' ""
+    cp 51-trezor.rules $out/lib/udev/rules.d/51-trezor.rules
+  '';
+
+  meta = with stdenv.lib; {
+    description = "udev rules for Trezor";
```

```suggestion
    description = "Udev rules for Trezor";
```

as per https://nixos.org/nixpkgs/manual/#chap-meta